includes/tracks/events/class-wc-orders-tracking.php

```diff
@@ -15,8 +15,81 @@ class WC_Orders_Tracking {
 	 * Init tracking.
 	 */
 	public function init() {
+		add_action( 'post_updated', array( $this, 'track_order_actions' ), 10, 2 );
+		add_action( 'woocommerce_new_order_item', array( $this, 'track_edit_order_add_item' ), 10, 3 );
 		add_action( 'woocommerce_order_status_changed', array( $this, 'track_order_status_change' ), 10, 3 );
 		add_action( 'load-edit.php', array( $this, 'track_orders_view' ), 10 );
+		add_action( 'load-post.php', array( $this, 'track_order_note_add' ), 10 );
+	}
+
+	/**
+	 * Send a Tracks event when an order line item is added.
+	 *
+	 * @param int    $item_id  Item id.
+	 * @param object $item  Item added to an order.
+	 */
+	public function track_edit_order_add_item( $item_id, $item ) {
+
+		$type = null;
+
+		if ( $item instanceof WC_Order_Item_Product ) {
+			$type = 'product';
+		} elseif ( $item instanceof WC_Order_Item_Fee ) {
+			$type = 'fee';
+		} elseif ( $item instanceof WC_Order_Item_Shipping ) {
+			$type = 'shipping';
+		} elseif ( $item instanceof WC_Order_Item_Tax ) {
+			$type = 'tax';
+		} elseif ( $item instanceof WC_Order_Item_Coupon ) {
+			$type = 'coupon';
+		}
+
+		if ( $type ) {
+			WC_Tracks::record_event(
+				'orders_edit_order_add_item',
+				array(
+					'type' => $type,
+				)
+			);
+		}
+	}
+
+	/**
+	 * Send a Tracks event when an order action is taken.
+	 */
+	public function track_order_actions() {
+		if ( ! empty( $_POST['wc_order_action'] ) ) { // @codingStandardsIgnoreLine
```

Comment:
Instead of using `@codingStandardsIgnoreLine` can we maybe use `// phpcs:ignore WordPress.Security.NonceVerification.NoNonceVerification`. This way PHPCS will only ignore one specific rule instead of all rules.